AdminUi/src/AdminUi/ClientApp/src/main.ts

```diff
@@ -4,4 +4,4 @@ import { AppModule } from "./app/app.module";
 
 platformBrowserDynamic()
     .bootstrapModule(AppModule)
-    .catch((err) => console.error(err));
+    .catch((err) => console.error(err)); // eslint-disable-line no-console
```

Comment:
Didn't we (you) introduce a logging library lately? Why don't you simply use that one instead of `console`? Same for all other occurrences of console.*

Reply:
This is an extension of that library. If you don't remember, we needed to include it in the logging library PR in order to make the local console logs have the same appearance as the endpoint logs.

Reply:
Hmm, I wasn't aware of that. Isn't it possible to actually use the logger of the library?

Reply:
Ahh, okay, then I'm fine with leaving it as it is.